vpp/vaapipostprocess_scaler.cpp

```diff
@@ -87,6 +88,18 @@ bool VaapiPostProcessScaler::getFilters(std::vector<VABufferID>& filters)
     return !filters.empty();
 }
 
+void VaapiPostProcessScaler::setRotationState(VAProcPipelineParameterBuffer* vppParam)
+{
+    if (m_transform != VPP_TRANSFORM_NONE) {
```

Comment:
We do not need this function.

L93 to L101 equals to one line
 vppParam->rotation_state = mapToVARotationState(m_transform);
You can add it to L147